Azure.Data.Wrappers/Interfaces.cs

```diff
@@ -422,6 +422,36 @@ public interface IQueueObject
         /// <param name="obj">Object</param>
         /// <returns>Task</returns>
         Task Send(object obj);
+
+        /// <summary>
+        /// Send a generic Object
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="obj"></param>
+        /// <returns></returns>
+        Task SendAsync<T>(T obj);
```

Comment:
I see some operations on these interfaces using the Async prefix and some not.  A little confusing to mix both together on async operations.  I generally use this approach, but would tend to stay away from it in this case as existing async operations don't seem to be following the pattern.  thoughts?

Reply:
Yes, it is unconsistent, but I'm having all new methods follow the convention. Granted it is confusing, but I wanted to do only non-breaking changes.

Reply:
Is the general convention in .NET to have the async suffix (even when there are no non-async methods). One point of this library was to 'force' async.

Reply:
good point @jefking .  i have referenced this in the past when this comes up.  https://stackoverflow.com/questions/15951774/does-the-use-of-the-async-suffix-in-a-method-name-depend-on-whether-the-async  

one part stuck out (not sure if this is applicable here):  

"there are always exceptions to a guideline. The most notable one in the case of naming would be cases where an entire type’s raison d’etre is to provide async-focused functionality, in which case having Async on every method would be overkill, e.g. the methods on Task itself that produce other Tasks."